File: app/src/main/java/com/base/base_app/data/model/GeneralResponseModel.java
package com.base.base_app.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GeneralResponseModel {
    @SerializedName("Code")
    @Expose
    private String code;
    @SerializedName("Title")
    @Expose
    private String title;
    @SerializedName("Message")
    @Expose
    private String message;
    @SerializedName("Success")
    @Expose
    private Boolean success;
    @SerializedName("Param")
    @Expose
    private Object param;
    @SerializedName("SeverityLevel")
    @Expose
    private Integer severityLevel;
    @SerializedName("Data")
    @Expose
    private Object data;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public Object getParam() {
        return param;
    }

    public void setParam(Object param) {
        this.param = param;
    }

    public Integer getSeverityLevel() {
        return severityLevel;
    }

    public void setSeverityLevel(Integer severityLevel) {
        this.severityLevel = severityLevel;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }
}

File: app/src/main/java/com/base/base_app/data/model/GeneralResponseModel.kt
package com.base.base_app.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GeneralResponseModel (
    @SerializedName("Code")
    @Expose
    var code: String? = null,

    @SerializedName("Title")
    @Expose
    var title: String? = null,

    @SerializedName("Message")
    @Expose
    var message: String? = null,

    @SerializedName("Success")
    @Expose
    var success: Boolean? = null,

    @SerializedName("Param")
    @Expose
    var param: Any? = null,

    @SerializedName("SeverityLevel")
    @Expose
    var severityLevel: Int? = null,

    @SerializedName("Data")
    @Expose
    var data: Any? = null
)